libafl/src/stages/pruning.rs

```diff
@@ -1,194 +0,0 @@
-//! Corpus pruning stage
```

Comment:
We ain't pruning no more?

Reply:
@tokatoka left a comment that it was not working anyways